sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -1322,7 +1323,7 @@ def replace_throughput(
         data = self.client_connection.ReplaceOffer(
             offer_link=throughput_properties[0]["_self"], offer=throughput_properties[0], **kwargs)
 
-        return ThroughputProperties(offer_throughput=data["content"]["offerThroughput"], properties=data)
+        return ThroughputProperties(offer_throughput=data["content"]["offerThroughput"], properties=data, headers=data)
```

Comment:
We're already passing `data` in to the `properties` parameter - do we need to be passing it twice?